Plugin/CppApi/include/ToolResourceProvider.h

```diff
@@ -74,14 +75,20 @@ public slots:
   void onMousePressedAndHeld(QMouseEvent& mouseEvent);
   void onMouseDoubleClicked(QMouseEvent& mouseEvent);
   void onIdentifyGraphicsOverlayCompleted(QUuid id, IdentifyGraphicsOverlayResult* identifyResult);
+  void onIdentifyGraphicsOverlaysCompleted(QUuid id, QList<IdentifyGraphicsOverlayResult*> identifyResults);
+  void onIdentifyLayerCompleted(QUuid taskId, IdentifyLayerResult* identifyResult);
+  void onIdentifyLayersCompleted(QUuid taskId, QList<IdentifyLayerResult*> identifyResults);
   void onScreenToLocationCompleted(QUuid taskId, const Point& location);
 
 signals:
   void sceneChanged();
   void geoViewChanged();
   void mapChanged();
   void spatialReferenceChanged();
-  void identifyGraphicsOverlayCompleted(QUuid id, IdentifyGraphicsOverlayResult* identifyResult);
```

Comment:
i don't think we want to change the signature of the signal, do we? Maybe leave this one and use `id` for all the others?